contracts.md

```diff
@@ -1,42 +1,42 @@
-# Contracts
+# واجهات كونتراكتس (Contracts)
```

Comment:
I think we need to remove all the translated words like "كونتراكس و فساد" because there is no real translation of it in the Arabic language, and it cannot be explained in just two words

Reply:
ok I'll leave those in English